packages/service-library/src/servicelib/aiohttp/requests_validation.py

```diff
@@ -115,7 +137,15 @@ def parse_request_query_parameters_as(
 ) -> ModelType:
     """Parses query parameters from 'request' and validates against 'parameters_schema'
 
-    :raises HTTPUnprocessableEntity (422) if validation of queries fail
+
+    Keyword Arguments:
```

Comment:
Q: do you use some extension to create these docstrings?

Reply:
yes. it is in the https://github.com/ITISFoundation/osparc-simcore/blob/c032fb5e133da4786d5384860daa4fecf672ab64/.vscode/extensions.json#L6

And check comments on pep257 in https://github.com/ITISFoundation/osparc-simcore/blob/c032fb5e133da4786d5384860daa4fecf672ab64/docs/coding-conventions.md#python